locale/en/docs/es6.md

```diff
@@ -55,12 +55,26 @@ All ES6 features are split into three groups for **shipping**, **staged**, and *
 
 * [Arrow Functions](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions)
 
+* [new.target](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/new.target)
+
+* [Object.observe](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/observe)
```

Comment:
Perhaps ES.Next or sth to that effect would be better; it's not final yet whether `Object.observe` will make it into 2016 AFAIK.